frontend/src/index.js

```diff
@@ -0,0 +1,51 @@
+let loggedInAccount
```

Comment:
same comment as before: show some love to ';'s

Reply:
I guess if we are going by standards rules unnecessary semicolons should not be written:
https://standardjs.com/rules.html